calendar-bundle/src/Resources/contao/modules/ModuleEventReader.php

```diff
@@ -14,8 +14,8 @@
 use Contao\CoreBundle\Exception\PageNotFoundException;
 use Contao\CoreBundle\Exception\RedirectResponseException;
 use Contao\CoreBundle\Image\Studio\Studio;
+use Contao\CoreBundle\Routing\ResponseContext\HtmlHeadBag\HtmlHeadBag;
```

Comment:
Is the double `\HtmlHeadBag\HtmlHeadBag` intended? It seems like the class `HtmlHeadBag` could also live in the `ResponseContext` namespace.

Reply:
It's intended because it will get more classes in the future for head tags.

Reply:
How will these future classes be called?

Reply:
Who knows? `Script`, `Link`, maybe?

Reply:
That sounds like the following to me:

```
src/Routing/ResponseContext/HtmlHeadBag.php
src/Routing/ResponseContext/HtmlHeadBag/Script.php
src/Routing/ResponseContext/HtmlHeadBag/Link.php
```

Reply:
I would like to keep the things inside their own namespace. Otherwise I have to add `JsonLdManager` inside `ResponseContext` as well. And then a potential `BodyTagBag` or what not as well and all of a sudden `ResponseContext` becomes cluttered.